cal/cal/static/js/services.js

```diff
@@ -0,0 +1,131 @@
+/*jslint devel: true */
+/*jslint browser: true */
+/*jslint jquery: true */
+/*global getCookie */
```

Comment:
I think it's important to us all to use a linter; it allows us to find syntax errors a lot faster, and small things like catching when we define a function parameter that we don't actually use. If you use a linter, these will prevent your linter from eternally yelling at you.

`devel` fixes the `console` being undefined
`browser` fixes `window` being undefined
`jquery` likewise
`getCookie` is defined in our legacy javascript code - it would be really nice to angularize it into a service, but I don't think that is worth the effort for now.

How about I combine the first three into one line?